src/v/net/probes.cc

```diff
@@ -110,6 +111,28 @@ void server_probe::setup_metrics(
       });
 }
 
+void server_probe::setup_public_metrics(
+  ss::metrics::metric_groups& mgs, const char* proto) {
+    namespace sm = ss::metrics;
+    auto aggregate_labels = config::shard_local_cfg().aggregate_metrics()
+                              ? std::vector<sm::label>{sm::shard_label}
+                              : std::vector<sm::label>{};
```

Comment:
`/public_metrics` is always aggregated, the flag is intended for the original `/metrics` endpoint.

Reply:
Done.